src/js/locale/it.yaml

```diff
@@ -139,14 +147,20 @@ services:
     title: Snapcast
   genius:
     title: Genius
-    switch_lyrics_result: Consulta altri risultati per i testi 
+    switch_lyrics_result: Consulta altri risultati per i testi
     want_lyrics: 'Desideri i testi delle tracce? Autorizza Genius '
   google:
     title: Google
   lastfm:
     title: LastFM
     love: Mi piace
     unlove: Non mi piace
+  youtube:
+    title: YouTube
+  tidal:
+    title: Tidal
+  jellyfin:
+    title: Jellyfin

```

Comment:
You don't technically need to provide these, as the translation will use English as a fallback (which have these same values)